src/OrchardCore.Modules/OrchardCore.Settings/ResourceConfigureOptions.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using OrchardCore.ResourceManagement.Core;
+
+namespace OrchardCore.Settings;
+
+public class ResourceConfigureOptions : IConfigureOptions<ResourceOptions>
+{
+    private readonly ISiteService _siteService;
+    private readonly IHttpContextAccessor _httpContextAccessor;
```

Comment:
You can use `HttpContext` here instead

Reply:
@hishamco not sure where would `HttpContext` would come from